unit/memory-analyzer/gdb_api.cpp

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************\
+
+Module: GDB Machine Interface API unit tests
+
+Author: Malte Mues <mail.mues@gmail.com>
+        Daniel Poetzl
+
+\*******************************************************************/
+
+#include <testing-utils/use_catch.h>
+
+#ifdef __linux__
```

Comment:
Anything Unix-y I'd have thought? Are you doing anything tricky enough that Cygwin wouldn't support it?

Reply:
Yes, changed to allow various Unices.